merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/generator/MissionModelGenerator.java

```diff
@@ -612,24 +612,18 @@ private static TypeSpec generateActivityTypesBuilder(final MissionModelRecord mi
   private static MethodSpec generateActivityTypesRegisterMethod(final MissionModelRecord missionModel) {
     final var typeName = missionModel.getTypesName();
 
-    final var cb = CodeBlock.builder()
-        .add(
-            "return new $T(new $T()",
-            typeName,
-            typeName.nestedClass("Builder"));
-
+    final var cb = CodeBlock.builder();
```

Comment:
I'd love a better name than `cb` 😺

Reply:
Haha yeah this is tricky. It's an `ActivityTypes.Builder` code block builder 🥴 Open to suggestions.

Reply:
Maybe just `builderChain`

Reply:
Yeah, `activityTypesBuilderChain` if we feel like being verbose. (Honestly, in this kind of templating setup, I think being verbose with names is a very good thing.)

Reply:
Nice, just pushed!